### PR TITLE
Covered the release of ember-stereo

### PR DESCRIPTION
**Pull request by ijlee2**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

**Comment by ijlee2** [comment]
@jkeen If there's additional information that you want to provide, feel free to make a suggestion.

File: content/the-ember-times-issue-188.md [hunk 1]
```diff
@@ -28,15 +30,19 @@ Twitter Thread: Ember is Data First 🐦,
...
 ---
 
-## [🐹 2. Section title in sentence case](section-url)
+## [Handle audios in Octane with `ember-stereo` 🎵](https://github.com/jkeen/ember-stereo)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Since 2016, [Jeff Keen (@jkeen)](https://github.com/jkeen) helped develop the open-source [ember-hifi](https://github.com/nypublicradio/ember-hifi) for the NY Public Radio.
```
Comment (jkeen) [inline content/the-ember-times-issue-188.md]: > for the NY Public Radio.

I think this should just read "for New York Public Radio"  for clarity

File: content/the-ember-times-issue-188.md [hunk 2]
```diff
@@ -28,15 +30,19 @@ Twitter Thread: Ember is Data First 🐦,
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+<!-- alex ignore retext-equality -->
+Jeff recently released its modern, reactive successor [ember-stereo](https://github.com/jkeen/ember-stereo). He has worked hard on making audio manipulation simple and providing [extensive examples](https://ember-stereo.com/docs). We encourage you to have a look!
+
+Out of the box, you will be provided with:
+
+- `stereo` service: It manages loading and playing sounds, making sure that only one sound plays at a time, setting volume, and providing system-level events that your application can tie into.
+- Several template helpers, to perform an action on, or retrieve information from, a sound or `url` string
```
Comment (jkeen) [inline content/the-ember-times-issue-188.md]: ```Template helpers that can perform an action on or retrieve information from a sound or `url` string (something entirely new from how `ember-hifi` did things!)```

File: content/the-ember-times-issue-188.md [hunk 3]
```diff
@@ -12,6 +13,7 @@ tags:
 👋 Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Handle audios in Octane with `ember-stereo` 🎵,
 Twitter Thread: Ember is Data First 🐦,
```
Comment (jkeen) [inline content/the-ember-times-issue-188.md:17]: I think this should be 
> reactively handle audio in Octane with `ember-stereo`

**Comment by jkeen** [comment]
@ijlee2 Thanks Isaac!

File: content/the-ember-times-issue-188.md [hunk 4]
```diff
@@ -12,6 +13,7 @@ tags:
 👋 Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Reactively handle audios in Octane with `ember-stereo` 🎵,
```
Comment (amyrlam) [inline content/the-ember-times-issue-188.md]:
```suggestion
Reactively handle audio in Octane with `ember-stereo` 🎵,
```

**Review by amyrlam — APPROVED** [review]
🙇‍♀️ thank you both, sounds super interesting!

**Pull request merged**